documentation/release_notes.rst

```diff
@@ -8,6 +8,61 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.5
+=============
+
+New Features
+------------
+- Added new random number distributions: ``exponential_distribution``, ``bernoulli_distribution``,
+  ``geometric_distribution``, ``lognormal_distribution``, ``weibull_distribution``, ``cachy_distribution``, ``extreme_value_distribution``.
+- Added the serial-based versions of the following algorithms: ``all_of``, ``any_of``, 
+  ``none_of``, ``count``, ``count_if``, ``for_each``, ``find``, ``find_if``, ``find_if_not``.
+  For the detailed list, please refer to `Tested Standard C++ API Reference`_. 
```

Comment:
LGTM aside from a couple nits. 
I am not able arrive at correct position. Maybe use `Tested Standard C++ API Reference <https://github.com/oneapi-src/oneDPL/blob/main/documentation/library_guide/tested_standard_cpp_api.rst#tested-standard-c-api-reference>`_ instead.
Please see https://github.com/haonanya/oneDPL/commit/a33b9ddee9fed74baf6dab463dd8edeb5d3b03d3 for details.
If there is something wrong, please correct me.
Thanks very much.

Reply:
Link is updated to https://github.com/oneapi-src/oneDPL/blob/release/2021.5/documentation/library_guide/tested_standard_cpp_api.rst#tested-standard-c-api-reference%60_..